### PR TITLE
feat(plasma-icons): add "Menu" icon

### PR DESCRIPTION
**Pull request by Yakutoc**
Необходимо добавить иконку "бургер" меню.  

#### Стало:

Снимок экрана 2023-03-28 в 12 55 21

:baby_chick: Download canary assets:

[colors--canary.448.4541858589.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.448.4541858589.xml)  
[color_metro_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.448.4541858589.swift)  
[color_metro_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.448.4541858589.kt)  
[color_metro_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.448.4541858589.ts)  
[color_sberHealth_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.448.4541858589.swift)  
[color_sberHealth_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.448.4541858589.kt)  
[color_sberHealth_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.448.4541858589.ts)  
[color_sbermarket_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.448.4541858589.swift)  
[color_sbermarket_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.448.4541858589.kt)  
[color_sbermarket_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.448.4541858589.ts)  
[color_sberprime_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.448.4541858589.swift)  
[color_sberprime_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.448.4541858589.kt)  
[color_sberprime_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.448.4541858589.ts)  
[color_selgros_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.448.4541858589.swift)  
[color_selgros_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.448.4541858589.kt)  
[color_selgros_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.448.4541858589.ts)  
[color_smbusiness_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.448.4541858589.swift)  
[color_smbusiness_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.448.4541858589.kt)  
[color_smbusiness_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.448.4541858589.ts)  
[PlasmaTokensColor--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.448.4541858589.swift)  
[shadow_sbermarket_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.448.4541858589.ts)  
[typo_mage_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.448.4541858589.swift)  
[typo_mage_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.448.4541858589.kt)  
[typo_mage_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.448.4541858589.ts)  
[typo_plasma_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.448.4541858589.swift)  
[typo_plasma_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.448.4541858589.kt)  
[typo_plasma_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.448.4541858589.ts)  
[typo_ruler_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.448.4541858589.swift)  
[typo_ruler_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.448.4541858589.kt)  
[typo_ruler_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.448.4541858589.ts)  
[typo_sage_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.448.4541858589.swift)  
[typo_sage_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.448.4541858589.kt)  
[typo_sage_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.448.4541858589.ts)  
[typo_sbermarket_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.448.4541858589.swift)  
[typo_sbermarket_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.448.4541858589.kt)  
[typo_sbermarket_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.448.4541858589.ts)  
[typo_soulmate_ios-swift--canary.448.4541858589.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.448.4541858589.swift)  
[typo_soulmate_kotlin--canary.448.4541858589.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.448.4541858589.kt)  
[typo_soulmate_react-native--canary.448.4541858589.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.448.4541858589.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.168.0-canary.448.4541858589.0
  npm install @salutejs/plasma-hope@0.26.0-canary.448.4541858589.0
  npm install @salutejs/plasma-icons@1.135.0-canary.448.4541858589.0
  npm install @salutejs/plasma-temple@1.146.0-canary.448.4541858589.0
  npm install @salutejs/plasma-ui@1.179.0-canary.448.4541858589.0
  npm install @salutejs/plasma-web@1.193.0-canary.448.4541858589.0
  # or 
  yarn add @salutejs/plasma-b2c@1.168.0-canary.448.4541858589.0
  yarn add @salutejs/plasma-hope@0.26.0-canary.448.4541858589.0
  yarn add @salutejs/plasma-icons@1.135.0-canary.448.4541858589.0
  yarn add @salutejs/plasma-temple@1.146.0-canary.448.4541858589.0
  yarn add @salutejs/plasma-ui@1.179.0-canary.448.4541858589.0
  yarn add @salutejs/plasma-web@1.193.0-canary.448.4541858589.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-448/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-448/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-448/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-448/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-448/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-448/temple-storybook/

File: packages/plasma-icons/src/Icon.assets/Menu.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { IconProps } from '../IconRoot';
+
+export const Menu: React.FC<IconProps> = (props) => (
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+        <path
+            d="M4.125 18.375H19.875M4.125 12.375H19.875M4.125 6.375H19.875"
+            stroke="currentColor"
+            strokeOpacity="0.96"
```
Comment (Yeti-or) [inline packages/plasma-icons/src/Icon.assets/Menu.tsx:10]: Давай прозрачность в иконке не хранить

File: packages/plasma-icons/src/Icon.assets/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { IconProps } from '../IconRoot';
+
+export const Menu: React.FC<IconProps> = (props) => (
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```
Comment (neretin-trike) [inline packages/plasma-icons/src/Icon.assets/Menu.tsx:6]: ширина тоже должна width="100%" быть

File: packages/plasma-icons/src/Icon.assets/Menu.tsx (hunk 1 above)
Comment (Yakutoc) [reply packages/plasma-icons/src/Icon.assets/Menu.tsx:10]: В проекте есть места, где используется **strokeOpacity**.

Снимок экрана 2023-03-28 в 16 57 20

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-448/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-448/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-448/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-448/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-448/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-448/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-448/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by neretin-trike — APPROVED** [review]

**Pull request merged**